angular.json

```diff
@@ -27,17 +27,17 @@
               "src/assets",
               {
                 "glob": "**/**/**/**/**/*",
-                "input": "projects/dg-docassembly-webcomponent/node_modules/@hmcts/document-viewer-webcomponent/assets",
+                "input": "dist/dg-docassembly-webcomponent/assets",
```

Comment:
does assets folder not get copied when you build the package. i feel like the path should not be changes to dist/... However, not sure if this is right.